eng/common/pipelines/templates/jobs/sdk-generation.yml

```diff
@@ -63,10 +54,12 @@ jobs:
           Write-Host "SpecRepoDirectory variable set to: $specRepoDirectory"
         }
 
-        if ('${{ parameters.SdkRepoUrl }}' -match $urlPattern) {
-          $sdkRepoOwner = $Matches['organization']
-          Write-Host "##vso[task.setvariable variable=SdkRepoOwner]$sdkRepoOwner"
-          Write-Host "SdkRepoOwner variable set to: $sdkRepoOwner"
+        if ('$(SdkRepoUrl)' -match $urlPattern) {
```

Comment:
We should still make the SdkRepoUrl a parameter and not expect it to be set as a variable. That make it more explicit. If someone wants to use a variable they can pass it via the parameter.

Reply:
I think the idea here is to make it so that it cannot be changed a queue time. Basically, we don't want pipeline for one language generating for another language because of user error. We are adding a parameter that allows user to change the repo org if they want to run from their fork. But the repo name will be hard set in the pipeline and cannot be changed without editing and modifying the pipeline.